### PR TITLE
Update id.m3u

### PR DESCRIPTION
**Pull request by F640**
- Added and edited some channels

### CODE REVIEW

File: streams/id.m3u [hunk 1]
```diff
@@ -206,9 +206,9 @@ https://stream.jogjatv.co.id/jtvlive/stream/index.m3u8
 http://122.248.43.138:1935/ch6/myStream/playlist.m3u8
 #EXTINF:-1 tvg-id="JTV.id" status="online",JTV (720p) [Not 24/7]
 http://122.248.43.138:1935/ch2/myStream/playlist.m3u8
-#EXTINF:-1 tvg-id="JTV.id" status="blocked",JTV (1080p)
-https://cdn02.mola.tv/Content/HLS/Live/channel(f2cd666d-f350-1140-7ce5-3d7f39f84727)/08.m3u8
-#EXTINF:-1 tvg-id="JTV.id" status="blocked",JTV (1080p) [Not 24/7]
+#EXTINF:-1 tvg-id="JTV.id" status="blocked",JTV (1080p) [Geo-blocked]
+https://cdn01.mola.tv/Content/HLS/Live/channel(f2cd666d-f350-1140-7ce5-3d7f39f84727)/index.m3u8
```
Comment (LaneSh4d0w) [inline streams/id.m3u:210]: 2022-05-04 data on stream, stops after a 10 second video.

File: streams/id.m3u [hunk 2]
```diff
@@ -314,6 +314,8 @@ http://210.210.155.37/uq2663/h/h143/index.m3u8
 http://122.248.43.138:1935/ch16/myStream/playlist.m3u8
 #EXTINF:-1 tvg-id="PalembangTV.id" status="online",Pal TV (720p) [Not 24/7]
 https://v3.siar.us/paltv/live/playlist.m3u8
+#EXTINF:-1 tvg-id="PalembangTV.id" status="online",Pal TV (720p) [Not 24/7]
+http://122.248.43.138:1935/ch7/myStream/playlist.m3u8
```
Comment (LaneSh4d0w) [inline streams/id.m3u:318]: I suggest you play with the number next to the "ch". You might have some surprises.

Comment (F640) [reply streams/id.m3u:316]: Played with it some time ago. Nothing special with it and some streams in this IP have inconsistent uptime.

Extra info: These streams from this IP also supports mpd (broken), rtmp, and rtsp format.

**Review by BellezaEmporium — DISMISSED** [review]

**Review by its0x08 — DISMISSED** [review]

**Review by ghost — APPROVED** [review]

**Review by BellezaEmporium — APPROVED** [review]

**Comment by F640** [comment]
Nothing special with it. I've checked it some time ago.

Pada tanggal Rab, 15 Jun 2022 14.07, LaneSh4d0w ***@***.***>
menulis:

**Pull request merged**